requirements.txt

```diff
@@ -5,8 +5,9 @@ jsonargparse[signatures]>=4.32.0
 lightning>=2.4.0
 parameterized>=0.9.0
 pytest>=7.4.4
-transformers>=4.44.0
+transformers>=4.49.0
 torch>=2.4.0
 torchmetrics>=1.4.0.post0
 twine>=4.0.2
-wandb>=0.18.0
\ No newline at end of file
+wandb>=0.18.0
+accelerate>=1.6.0
```

Comment:
sort lexicographically. also if this is a strict dependency it needs to be declared in `pyproject.toml` too. i would suggest testing (locally or on circleCI) without it and see if just upgrading `transformers` is enough.